src/stylesheets/components/_navbar.scss

```diff
@@ -9,9 +9,18 @@
   @include grid-container($theme-header-max-width);
 }
 .sds-navbar__title{
-  
   @include u-font('heading','md') ;
   @include u-text('semibold') ;
- 
+}
+
+.sds-navbar-blank {
+  min-height: 3rem;
+}
+
+.sds-nav__secondary-blank {
+  bottom: 3rem;
+}
```

Comment:
Is blank a variant of sds-navbar? 

If so the BEM naming format would be:

.sds-navbar--blank
and
.sds-nav__secondary--blank

http://getbem.com/

Reply:
Updated to match naming convention